lib/index.js

```diff
@@ -3,27 +3,41 @@ var $ = require('jquery');
 var gameStart = document.getElementById("game-start");
 var Game = require('./game.js');
 var levelUpScreen = document.getElementById("level-up");
-var fillCanvas = document.getElementById('scoresCanvas')
+var fillCanvas = document.getElementById('scoresCanvas');
+var game;
 
-requestAnimationFrame(function(){
+setUpStartScreen();
+
+function setUpStartScreen(){
   $("#game-div").hide();
   $("#game-over").hide();
   $("#instructions").hide();
   $(levelUpScreen).hide();
-  $('#start-btn').on('click', function(){
-    $("body").css("background-color", "#F5F3EE");
-    $(gameStart).hide();
-    $("#game-div").show();
-    var game = new Game(canvas, fillCanvas);
-    game.start();
-  });
+}
```

Comment:
Much better. Should we consider pulling more of this out? Should we assign these jQuery objects to variables